guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -1,24 +1,79 @@
 [id="preparing-a-host-for-external-databases_{context}"]
 = Preparing a host for external databases
 
-Install a freshly provisioned system with the latest {EL} 8 to host the external databases.
+Install a freshly provisioned system with the latest {EL} 9 or {EL} 8 to host the external databases.
 
 ifdef::satellite[]
 Subscriptions for {RHEL} do not provide the correct service level agreement for using {Project} with external databases.
 You must also attach a {Project} subscription to the base operating system that you want to use for the external databases.
 endif::[]
 
+ifdef::satellite[]
+Use the instructions in {InstallingServerDocURL}attaching-infrastructure-subscription_{project-context}[Attaching the {Project} Infrastructure Subscription] to attach a {Project} subscription to your server.
+endif::[]
+
 .Prerequisites
 * The prepared host must meet {Project}'s {InstallingServerDocURL}storage-requirements_{project-context}[Storage Requirements].
 
 .Procedure
 ifndef::orcharhino[]
-. Use the instructions in {InstallingServerDocURL}attaching-infrastructure-subscription_{project-context}[Attaching the {Project} Infrastructure Subscription] to attach a {Project} subscription to your server.
-. Disable all repositories and enable only the following repositories:
+Select the operating system and version you are installing external database on:
+
+* xref:#repositories-externaldb-el-9[{EL} 9]
+* xref:#repositories-externaldb-el-8[{EL} 8]
+
+[id="repositories-externaldb-el-9"]
+== {EL} 9
```

Comment:
You can make this `RHEL` because the instructions only apply to RHEL.
```suggestion
== {RHEL} 9
```

Reply:
It would also apply to a host such as AlmaLinux 9 that is connected to Foreman. Does "external" mean your DB host is not a host managed by Foreman?

Reply:
I would keep this unchanged as it is not downstream-only.

Reply:
> It would also apply to a host such as AlmaLinux 9 that is connected to Foreman. Does "external" mean your DB host is not a host managed by Foreman?

Managed host (DB host) in this case if I am not wrong.

Reply:
> I would keep this unchanged as it is not downstream-only. But, I am not sure about the external DB support to the upstream builds.

https://github.com/theforeman/foreman-documentation/pull/2396 was the effort I started to fix it for upstream.

> Managed host (DB host) in this case if I am not wrong.

The DB host doesn't have to managed by Foreman. Users need to make sure there's a PostgreSQL connection that works and that's it. How they achieve that, is up to them. They can also build complex solutions, such as a load balanced multi-host setup, though we don't document that today.